documentation/book/proc-renewing-ca-certs-manually.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-renewing-ca-certs-manually-{context}']
+
+= Renewing CA certificates manually
+
+Unless the `Kafka.spec.clusterCa` and `Kafka.spec.clientsCa` objects are set to `false`, the cluster and clients CA certificates will auto-renew at the start of their respective certificate renewal periods. 
+You can manually renew one or both of these certificates before the certificate renewal period starts, if required for security reasons. 
+A renewed certificate uses the same private key as the old certificate.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+* One or more maintenance time windows are configured for the Kafka cluster.
```

Comment:
Actually, you can avoid having a maintenance time window. It just means that the renew will be done during the next reconciliation.